mlrun/api/launcher.py

```diff
@@ -12,9 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 import typing
+from typing import Dict, List, Union
```

Comment:
redundant, and also we prefer using the whole namespacing of the imported package, like above

Reply:
discussed offline, using 'from' just for typing